### PR TITLE
fix(docs): remove subfolders because we use relative linking

### PR DESCRIPTION
**Pull request by raisedadead**
This is in preparation for addressing #39785

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi @Sky020,

So, the issue with the current setup on master is:

1. We need to use [relative linking](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/index.html#L57) on pages. This is so that we render the corresponding versions of the sidebar and the links in them. Docsify renders the sidebar it sees in the folder, i.e. it will render the `i18n/Chinese/_sidebar.md` when we are in `i18n/Chinese/` folder, and so on.

2. But the above also means we are unable to use subfolders for other things without duplicating the sidebar in those as well. So, if we had a `flight-manuals` subfolder we would need a sidebar in that too for relative linking to play nicely. It doesn't at the minute and that is the bug.

To address this, in this PR I am consolidating the documents a bit and using only the root. This doesn't fix the bug as of yet, but will when the changes cascade into the translations after this PR is merged and we sync up via Crowdin.

Please review and merge.

**Review by ShaunSHamilton — COMMENTED** [review]
This makes a lot more sense, because something like `devops` was under _flight-manuals_ but did not follow the same directory structure. I am not sure how that even happens - That docsify knew to add it under _flight-manuals_.

@raisedadead Are you sure about the change to the _Chinese_ section?

Would that not still lead to an issue like:
```text
https://contribute.freecodecamp.org/#/i18n/Chinese/i18n/Chinese/news-author-application
```

Should this not also be consolidated? Perhaps to the `/i18n/Chinese` translation only?

**Comment by raisedadead** [comment]
> I am not sure how that even happens - That docsify knew to add it under flight-manuals.

It's just how we linked files in the `_sidebar.md` (some using a subfolder, and others did not) and I agree it was a bit convoluted. Anyways, it should be straight now.

> Are you sure about the change to the Chinese section?

That's a good catch. They should remain direct, should be fixed now.

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
 
+<!-- The section below should not use relative linking -->
+
 - **中文社区贡献指南**
-  - [成为专栏作者](/i18n/chinese/news-author-application.md)
-  - [文章翻译计划](/i18n/chinese/news-translations.md)
-  - [视频翻译计划](/i18n/chinese/video-translations.md)
+  - [成为专栏作者](/i18n/Chinese/news-author-application.md)
+  - [文章翻译计划](/i18n/Chinese/news-translations.md)
+  - [视频翻译计划](/i18n/Chinese/video-translations.md)
```
Comment (Sky020) [inline docs/_sidebar.md:22]: This is now the only section with `Chinese` instead of `chinese`

I assume all the rest should be capital `C`

Comment (raisedadead) [reply docs/_sidebar.md:22]: Yeah, that should be fixed when we sync-up with Crowdin.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍  Congrats on your PR, Mrugesh. 

We look forward to your future contribution 😉

**Pull request merged**